src/Assets/Animations/AnimationCustom.cpp

```diff
@@ -8,16 +8,18 @@
 #include <iostream>
 #include "AnimationNode.h"
 
-Transformation AnimationCustom::calculateTransform(const std::string& nodeName __attribute((unused)), float time, bool &isFound) const {
-    Transformation resultTransformation;
-
-    resultTransformation.setScale( animationNode->getScalingVector(time));
-    resultTransformation.setTranslate( animationNode->getPositionVector(time));
-    resultTransformation.setOrientation(animationNode->getRotationQuat(time));
+bool AnimationCustom::calculateTransform(const std::string& nodeName __attribute((unused)), float time, Transformation& transformation) const {
+    bool status = false;
```

Comment:
Since status is not a parameter, you can just return it, without creating a variable